server/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:12.18.1
```

```suggestion
FROM node:14.16.1-buster-slim
```
LTS version has change since you've open this PR